### ISSUE TITLE
[question] URL to set options?

### ISSUE THREAD

**Issue by l3iggs** [AHAAAAAAA/PokemonGo-Map#1823]
Is there a way to define filters/alerts and turn on the alarm function via the url?
Something like:
`http://localhost:5000/?sound=true&notify=pidgey&notify=weedle&filter=snorlax&filter=golem`
(If I want sound alerts for pidgey and weedle and I don't want snorlax or golem to show up on the map)

**Comment by chuyskywalker** [comment]
No, not right now. All those options are saved in your browsers `localstorage` not server-side.

**Comment by l3iggs** [comment]
Thanks @chuyskywalker 
Is do you know if there is any way to quickly and easily populate the filter and notify pokemon fields without manually entering the names?

**Comment by chuyskywalker** [comment]
You could probably hack together some kind of javascript command that sets all the localstorage values directly and just copy/paste that into a console.

**Comment by l3iggs** [comment]
Thanks for the idea @chuyskywalker, here's the javascript that goes with my above example:
```javascript
localStorage.setItem('playSound', "true");
localStorage.setItem('remember_select_exclude', "[76,143]");
localStorage.setItem('remember_select_notify', "[13,16]");
```

**Comment by l3iggs** [comment]
Using the URL to set these options would be waaay better though :-)
